.github/workflows/master.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: master
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn
+      - run: yarn test
+      - run: yarn lint
+      - run: yarn build
+      # TODO: Update the deployment scripts from here
+      # Add appropriate secrets
+      # I could update to heroku using actions for testing: https://radiant-chamber-05759.herokuapp.com/
+      - name: Deploy to heroku
+        uses: akhileshns/heroku-deploy@v3.5.7
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: ${{secrets.HEROKU_APP_NAME}}
+          heroku_email: ${{secrets.MY_EMAIL}}
```

Comment:
@subash-poudel We've been deploying to our AWS s3 bucket which is pointed to https://frogtoberfest.lftechnology.com. It needs to be the same. Check the old travis script for the details.

Can you make this change? You may check with @sparshadotel for help if needed.

Reply:
Or, we may just change the deployment to AWS Amplify instead of s3 bucket. This might be even easier.

Reply:
I have worked on amplify and can deploy to amplify if necessary. Need some help with the url stuffs for amplify.